manifests/8/86Box/86BoxManager/1.7.4/86Box.86BoxManager.locale.en-US.yaml

```diff
@@ -5,7 +5,7 @@ Publisher: 86Box Project
 PublisherUrl: https://86box.net
 PublisherSupportUrl: https://paypal.me/86Box
```

Comment:
I know it wasn't part of the initial change, but this doesn't look like a valid support URL

Reply:
I've added `paypal` to my "word filter list", to help mitigate this in the future.

Reply:
Hi @Dragon1573,

Just commenting out the `PublisherSupportUrl` would probably be enough here.

Reply:
```suggestion
# PublisherSupportUrl:
```